R/extend_family.R

```diff
@@ -14,6 +14,45 @@
 #' [init_refmodel()], so you will rarely need to call it yourself.
 #'
 #' @param family An object of class `family`.
+#' @param latent A single logical value indicating whether to use the latent
+#'   projection (`TRUE`) or not (`FALSE`). Note that setting `latent = TRUE`
+#'   causes all arguments starting with `augdat_` to be ignored.
+#' @param latent_y_unqs Only relevant for a latent projection where the original
+#'   response space has finite support (i.e., the original response values may
+#'   be regarded as categories), in which case this needs to be the character
+#'   vector of unique response values (which will be assigned to `family$cats`
+#'   internally) or may be left at `NULL` (so that \pkg{projpred} will try to
+#'   infer it from `family$cats`). See also section "Latent projection" below.
+#' @param latent_ilink Only relevant for the latent projection, in which case
+#'   this needs to be the inverse-link function. If the original response family
+#'   was the [binomial()] or the [poisson()] family, then `latent_ilink` can be
+#'   `NULL`, in which case an internal default will be used. Can also be `NULL`
+#'   in all other cases, but then an internal default based on `family$linkinv`
+#'   will be used which might not work for all families. See also section
+#'   "Latent projection" below.
+#' @param latent_llOrig Only relevant for the latent projection, in which case
```

Comment:
I wanted to avoid the suffix `_orig` as it's already used at other places in the code. (So using `Orig` guaranteed that all objects related to response-scale analyses can be found by searching for `Orig`.) Would the suffix `_ors` be ok for you (for *original response scale*)? That would still be unique and would also have the same number of characters; otherwise, I would have to re-indent affected code (and re-flow to margin width 80).

Reply:
As discussed internally, I have changed this to `_oscale`.